x-pack/plugins/spaces/server/saved_objects/spaces_saved_objects_client.ts

```diff
@@ -397,9 +401,16 @@ export class SpacesSavedObjectsClient implements SavedObjectsClientContract {
     options: SavedObjectsOpenPointInTimeOptions = {}
   ) {
     throwErrorIfNamespaceSpecified(options);
+
+    const namespaces = await this.getSearchableSpaces(options.namespaces);
+    if (namespaces.length === 0) {
+      // throw bad request if no valid spaces were found.
+      throw SavedObjectsErrorHelpers.createBadRequestError();
+    }
+
     return await this.client.openPointInTimeForType(type, {
       ...options,
-      namespace: spaceIdToNamespace(this.spaceId),
+      namespaces,
```

Comment:
The `find` wrapper is not using `spaceIdToNamespace`, so I guessed I also don't need it for `openPointInTimeForType`, but would like a confirmation

Reply:
Yep this is correct